src/pages/documentation/iam/public-classes.md

```diff
@@ -1,6 +1,6 @@
 import Tabs from './tabs/public-classes.md'
 
-# Class - Message
+# iOS Class - Message
```

Comment:
Is this iOS only for now? Or is this something that's iOS only forever?

Reply:
the public facing Message class is now iOS only. the Android side now has a public facing interface instead.